overlord/ifacestate/handlers.go

```diff
@@ -487,7 +490,7 @@ func (m *InterfaceManager) doDisconnect(task *state.Task, _ *tomb.Tomb) error {
 		}
 	}
 
-	cref := interfaces.ConnRef{PlugRef: plugRef, SlotRef: slotRef}
+	remapOutgoingConnRef(st, cref)
```

Comment:
should we have getConn/setConn singular that do the mapping inside?

Reply:
Yeah, perhaps. I'll look into it